README.md

```diff
@@ -63,16 +64,15 @@ lr_finder.plot(log_lr=False)
 - The learning rate and loss history can be accessed through `lr_finder.history`. This will return a dictionary with `lr` and `loss` keys.
 - When using `step_mode="linear"` the learning rate range should be within the same order of magnitude.
 
-
 ## Additional support for training
 
 ### Gradient accumulation
 
-You can use `AccumulationLRFinder` to find learning rate with the mechanism of gradient accumulation.
+You can use `LRFinder` to find learning rate with the mechanism of gradient accumulation.
```

Comment:
Since `AccumulationLRFinder` is integrated into `LRFinder`, would it be better to change this line to the following one?
```
You can set `accumulation_steps` with a proper value to perform gradient accumulation.
```

Reply:
Agreed